apps/examples/testcase/le_tc/network/tc_net_dup.c

```diff
@@ -107,6 +107,7 @@ static void tc_net_dup2_p(void)
 	int sock_new = socket(AF_INET, SOCK_DGRAM, 0);
 	if (sock_new == -1) {
 		UTC_NET_LOGE("get new socket error (%d)\n", errno);
+		return;
```

Comment:
calling close(socket_old) is neccessary.

Reply:
ok